lib/tool_belt/commands/procedure.rb

```diff
@@ -0,0 +1,56 @@
+module ToolBelt
+  module Command
+    class ProcedureCommand < Clamp::Command
+      subcommand "foreman", "Foreman procedures" do
+        subcommand "branch", "Start branching Foreman" do
+          parameter "release", "Version that the branch will have - like 1.20"
```

Comment:
I don't think this is totally clear. This should be the version that the foreman currently has and then will be bumped? I guess I don't totally understand why the person entering the command doesn't bump it themselves.

Reply:
This is the procedure to branch nightly into 1.20 and then bump nightly to 1.21. In theory you could automatically detect it somewhere but I think that's out of scope for now.